spec/features/managing_step_by_step_pages_spec.rb

```diff
@@ -562,4 +564,8 @@ def there_should_be_no_unpublish_button
       expect(page).to_not have_css("button", text: "Unpublish step by step")
     end
   end
+
+  def then_I_can_preview_the_step_by_step
+    expect(page).to_not have_button("Preview")
```

Comment:
I'm a bit confused why we are expecting the page `to_not` have the "Preview" button if we want to be able to preview?

Reply:
Good spot!

Reply:
Why did the test pass though?

Reply:
I'm wondering that too.

Reply:
I think it's because "Preview" is classed as a "link" not a "button"

When I changed the test to `expect(page).to have_button("Preview")`, it failed, but `expect(page).to have_link("Preview")` works.

Reply:
Will update and fix up.